pkg/cidata/cidata.TEMPLATE.d/user-data

```diff
@@ -11,7 +11,7 @@ package_upgrade: true
 package_reboot_if_required: true
 {{- end }}
 
-{{- if or .RosettaEnabled (or (eq .MountType "9p") (eq .MountType "virtiofs")) }}
+{{- if or .RosettaEnabled (and .Mounts (or (eq .MountType "9p") (eq .MountType "virtiofs"))) }}
```

Comment:
I know this is pre-existing code, but I wonder if the condition should just exclude `reverse-sshfs` in case we add more mount types in the future:

```suggestion
{{- if or .RosettaEnabled (and .Mounts (ne .MountType "reverse-sshfs")) }}
```